triton/tut/interactive.rst

```diff
@@ -190,56 +188,58 @@ It's best to use smaller values when submitting interactive jobs, and more for b
 Exercises
 =========
 
-1. The program ``/scratch/scip/hpc-examples/slurm/memory-hog.py``
-   uses up a lot of memory to do nothing.  Let's play with it.  It's
-   run like this: ``python
-   /scratch/scip/hpc-examples/slurm/memory-hog.py 50M``, where the
-   last argument is however much memory you want to eat.  (also
-   available from `hpc-examples/slurm
-   <https://github.com/AaltoSciComp/hpc-examples/tree/master/slurm>`__)
+The scripts you need for the following exercises can be found here:
+`hpc-examples/slurm
+<https://github.com/AaltoSciComp/hpc-examples/tree/master/slurm>`__)
+You can clone the repository by running
+``git clone https://github.com/AaltoSciComp/hpc-examples.git``
 
-   a) Try running the program with ``50M``
+1. The program ``hpc-examples/slurm/memory-hog.py``
+   uses up a lot of memory to do nothing.  Let's play with it.
+   It's run as follows:
+   ``python hpc-examples/slurm/memory-hog.py 50M``, where the
+   last argument is however much memory you want to eat.
+
+   a) Try running the program with ``50M``.
 
    b) Run the program with ``50M`` and ``srun --mem=500M``.
 
-   c) Increase the amount of memory allocated until the job fails.
-      What happens?
+   c) Increase the amount of memory allocated.  How much memory can
+      you use before the job fails?
 
-   d) Play around with different parameters: how much memory can you
-      use?
+   d) Look at the job history using ``slurm history`` - can you see
+      how much memory it actually used? (Try using the ``--sleep``
+      option)
```

Comment:
```suggestion
   d) Look at the job history using ``slurm history`` - can you see
      how much memory it actually used?
      
      Note, Slurm only measures memory every 60 seconds or so.  To make the program last longer, so that can be measured, give the ``--sleep`` option to the Python process, like this: ``python memory-hog.py 50M --sleep=60``.
```
is this the right syntax for --sleep?

Reply:
Yes, that's it.